powerline/config_files/themes/powerline_unicode7.json

```diff
@@ -143,6 +143,15 @@
 			"args": {
 				"text": "🖫⃥"
 			}
+		},
+
+		"powerline.segments.i3wm.scratchpad": {
+			"args": {
+				"icons": {
+					"fresh": "O",
+					"changed": "X"
```

Comment:
`unicode7` has nothing to do with UTF-7, it is a file which may contain code points from unicode-7.0 standard.

Icons here *may* be the new ones introduced by the standard, not *must be*. If you do not know newer code points which are more relevant, then you should have characters from powerline theme here.